doc/rst/technotes/libraries.rst

```diff
@@ -591,3 +740,52 @@ supported.  We hope to extend this support in the future.
 
 C interoperability does not support default values for function arguments.  We
 do not anticipate supporting argument default values in C.
+
+.. _Python Intents:
+
+Intents in Python Interoperability
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Libraries compiled for Python do not support the default intent for ``string``,
+``c_string`` or 1D arrays, as copies are currently performed (which would
+violate ``const ref``).  Instead, only the ``in`` and ``const in`` intents are
+supported for these types.  We may be able to support all intents in the future.
+
+.. _Multilocale Caveats:
+
+Multilocale Caveats
+-------------------
+
+.. _Other Settings:
+
+Other Settings
+~~~~~~~~~~~~~~
+
+We intend to expand support for ``--library`` to include:
+
+- ``CHPL_COMM = ugni``
+- ``--no-local``
+- ``CHPL_COMM = none`` when ``CHPL_LAUNCHER != none``
+
+These settings would behave similarly to the current behavior with ``CHPL_COMM =
+gasnet``, when relevant - for instance, it is expected that all of these
+settings would result in an executable that communicates with the user's program
+via the generated library.
+
+Other configurations may also become supported in the future.
+
+Supported Types
+~~~~~~~~~~~~~~~
+
+Multilocale libraries support the same argument and return types as single
+locale libraries, with the notable exception of ``complex`` numbers, arrays, and
+pointer types.  We anticipate extending the supported types in the future,
+though may not end up supporting pointer types.
+
+Intents
+~~~~~~~
+
+Multilocale libraries do not support the default intent for ``string`` and
```

Comment:
Does this a apply to 1D arrays too like the python caveats mention above?

Reply:
1D arrays are not supported in multilocale libraries, but it probably will apply to them when they are supported